### DISCUSSION TITLE
【提案】1. 是否支持开设WTF html/css/js 极简教程

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#183] [Polls]
Web3中，区块链就是后端，那么前端技术栈就很重要了。这个投票是想调查WTF学院的成员是否支持开设WTF html/css/js 前端系列的极简教程。

好处：
- 教程更注重Dapp开发需要的内容，更快上手前端开发。

坏处：
- 目前中文社区已有很多前端教程，做这个系列可能会浪费时间。

截止日期：2022年9月5日24点。

**Comment by Lokiscripter** [comment]
可以做个资料汇总，放上已有的html/css/js甚至react，nodejs相关的链接，不必要做教程。

**Reply by AmazingAng** [reply]
嗯，要是不开设这个系列教程的话，就做资料汇总

**Reply by anbang** [reply]
> 可以做个资料汇总，放上已有的html/css/js甚至react，nodejs相关的链接，不必要做教程。

前端的资料无论书籍，文档，视频，面试题等，无论免费还是培训机构的付费内容，都已经非常完备和成熟；完全没有必要浪费精力做这些意义不是很大的事情。

**Comment by dethan3** [comment]
前端三剑客在互联网上已经有完备的教程了，无论是视频教程还是文档，我个人推荐MDN

**Comment by AzumWatson** [comment]
一楼二楼说的在理 赞同
